### PR TITLE
Introduce "want_keystone_v3" to allow keystone v3 testing

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1802,6 +1802,9 @@ function custom_configuration()
                 #zypper ar --refresh http://download.suse.de/ibs/SUSE:/CA/SLE_11_SP3/SUSE:CA.repo
                 #zypper -n --gpg-auto-import-keys in ca-certificates-suse
             fi
+            if [[ $want_keystone_v3 ]] ; then
+                proposal_set_value keystone default "['attributes']['keystone']['api']['version']" "'3'"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: `"'v3'"`?

Comment (rhafer) [inline scripts/qa_crowbarsetup.sh]: No. "3"  See: https://github.com/rhafer/barclamp-keystone/commit/7f78be5e733a60db35c9c7c51ddabe18da6a526a

**Comment by aplanas** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**